### ISSUE TITLE
Installation script not working on Windows

### ISSUE THREAD

**Issue by Trayani** [korlibs/korge#2294]
Hello, 
I wanted to check out this engine so I followed the online instructions to install it. 
But the script always fails. 
```

C:\Users\MyUser\Downloads>install-korge-forge.cmd
KorGE Forge Installer v2024.2.0.2-patch1
Downloading https://github.com/korlibs/universal-jre/releases/download/0.0.1/xzdec.exe into C:\Users\MyUser\.korge\xzdec.exe
The syntax of the command is incorrect.
```

I tinkered a bit with the script. It seems that the syntax error is coming from this part ↓↓↓.

```

    IF /i "%DOWNLOAD_SHA256%" == "%EXPECTED_SHA256%" (
        MOVE "%LOCAL_PATH%.tmp" "%LOCAL_PATH%" > NUL 2> NUL
    ) ELSE (
        ECHO ERROR downloading %URL%, SHA256=%DOWNLOAD_SHA256%, but expected SHA256=%EXPECTED_SHA256%
        EXIT /B -1
    )
```

 Once I removed, it, the script returned more output 

```

C:\Users\MyUser\Downloads>install-korge-forge.cmd
KorGE Forge Installer v2024.2.0.2-patch1
A subdirectory or file C:\Users\MyUser\Downloads\korge-forge-installer already exists.
Downloading https://github.com/korlibs/universal-jre/releases/download/0.0.1/xzdec.exe into C:\Users\Jan\.korge\xzdec.exe
Downloading https://github.com/korlibs/korge-forge-installer/releases/download/v2024.2.0.2-patch1/korge-forge-installer.jar into C:\Users\Jan\Downloads\korge-forge-installer\korge-forge-installer-v2024.2.0.2-patch1.jar
Downloading https://github.com/korlibs/universal-jre/releases/download/0.0.1/microsoft-jre-21.0.3-windows-x64.tar.xz into C:\Users\Jan\.korge\jre-21.tar.xz
Extracting C:\Users\MyUser\.korge\jre-21.tar.xz...
The system cannot find the file specified.
```

Obviously, I will not try to use the downloaded files since  authenticity was not verified.
But it prevents me from using the engine, hence the issue :)

**Comment by richard-porteous** [comment]
I have the same problem. windows 10 if I download the file first otherwise removing the s and using the full command.
PS C:\Users\ratpa\Downloads> cd "%USERPROFILE%/Downloads" & curl -LO https://forge.korge.org/install-korge-forge.cmd &&
install-korge-forge.cmd

Id     Name            PSJobTypeName   State         HasMoreData     Location             Command
--     ----            -------------   -----         -----------     --------             -------
3      Job3            BackgroundJob   Running       True            localhost            cd "%USERPROFILE%/Downlo…
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) schannel: next InitializeSecurityContext failed: CRYPT_E_NO_REVOCATION_CHECK (0x80092012) - The revocation function was unable to check revocation for the certificate.

**Comment by richard-porteous** [comment]
this is a show stopper

**Comment by richard-porteous** [comment]
this could be solved with manual instructions and a correct up-to-date sha README in the downloads folder
I got way further once I split out the commands from the script and install jre or jdk 21 for myself

**Comment by richard-porteous** [comment]
PS antivirus could get in the way - needs to be stopped during install

**Comment by richard-porteous** [comment]
OK - so update - I installed java JDK 21 first using the eclipse adoption installer, and now everything just works as it should, so that is where you should test your script is on windows 10 with no java installed

**Comment by Trayani** [comment]
Thank you for sharing your solution. 
I assume you meant [this](https://adoptium.net/) by _eclipse adoption installer_?  I installed the JDK (and changed my JAVA_HOME to it)  and restarted my PC to be sure.   I'm still getting the syntax error though.

**Comment by richard-porteous** [comment]
Yes. The cmd (install script) file ran once I rebooted after installing the eclipse adoption installer. 
However following the cmd script file hint get the installer jar file
https://github.com/korlibs/korge-forge-installer/releases/download/v2024.2.0.2-patch1/korge-forge-installer.jar
and then run that
